lib/shared/layers/python-sdk/python/genai_core/documents.py

```diff
@@ -190,9 +208,30 @@ def set_status(workspace_id: str, document_id: str, status: str):
             ":timestampValue": timestamp,
         },
     )
+    update_timestamp(workspace_id, document_id)
```

Comment:
the content of this function seems to be redundant with the above code. Was the idea to move the whole logic into the `update_timestamp` function?
the return value is not used, is this correct?

Reply:
You are correct. I removed `update_timestamp` function. The `update_subscription_timestamp` still exists and is leveraged throughout the code.